doc/api.rst

```diff
@@ -37,8 +37,8 @@ Graph protocols
 .. autosummary::
    :toctree: generated/
 
-   GraphProtocol
-   EquivalenceClassProtocol
+   Graph
+   EquivalenceClass
```

Comment:
I think the DCO rebasing makes these PR diffs nonsensical for some reason. E.g. this diff isn't technically part of this PR, but the previous PR... rip. idk how to fix (usually it is to rebase, but seeing as how you did..)